docs/pages/reference/configuration/environment-variables.mdx

```diff
@@ -217,6 +217,14 @@ Whether the ClickHouse user has read-only access or not.
 | --------------- | ---------------------- | --------------------- |
 | `true`, `false` | N/A                    | N/A                   |
 
+## `CUBEJS_DB_CLICKHOUSE_COMPRESSION`
+
+Whether the ClickHouse client has compression enabled or not.
+
+| Possible Values | Default in Development | Default in Production |
+| --------------- | ---------------------- | --------------------- |
+| `true`, `false` | N/A                    | N/A                   |
```

Comment:
Why `N/A`? I see that it's `false` by default, right?

Reply:
You are right, I updated the docs